packages/react-keytips/package.json

```
@@ -2,7 +2,11 @@
   "name": "@fluentui-contrib/react-keytips",
   "version": "0.0.1",
   "dependencies": {
-    "@swc/helpers": "~0.5.2"
+    "@swc/helpers": "~0.5.2",
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
+    "@fluentui/react-utilities": ">=9.16.0 < 10.0.0",
+    "@fluentui/react-positioning": ">=9.16.0 < 10.0.0",
```

We shouldn't use ranges in prod dependencies
```suggestion
    "@fluentui/react-positioning": "^9.16.0",
```

---

thx. will change it for react-positioning as u suggested, other deps with ranges are has to be fixed in other packages as well, otherwise CI will fail on syncpack check